doc/api/http.md

```diff
@@ -2451,6 +2451,9 @@ header name:
   `etag`, `expires`, `from`, `host`, `if-modified-since`, `if-unmodified-since`,
   `last-modified`, `location`, `max-forwards`, `proxy-authorization`, `referer`,
   `retry-after`, `server`, or `user-agent` are discarded.
+  To allow duplicate values of the headers listed above to be joined,
+  use the option `joinDuplicateHeaders` in [`http.request()`][]
+  and [`http.createServer()`][].
```

Comment:
It might be helpful to make a reference to the relevant RFC here (to clarify to readers that this is about standards compliance)?